### DISCUSSION TITLE
Add a new job, Telecomms Technician

### DISCUSSION THREAD

**Discussion by AtnerNT** [unitystation/unitystation#8185] [Ideas]
While telecoms is pretty fun, I feel in most rounds its usually forgotten about to a degree (I know we dont have telecomms in unitystation yet- this is just for the future). As such I would like to propose a minor expansion to telecomms and a new job to go along with it.

First off the comm tech will most likely function similarly to the atmos tech, but rather than specializing in piping and gases they would specialize in radios and frequencies, to make the role feel unique to your typical station engineer they would also most likely come with their own jumpsuit/skirt and hardsuit just like the atmos tech.

One major aspect that makes playing as an atmos tech or any other engineer enjoyable is the element of experimentation and just working on projects. Station engineers may expand the station while atmos techs may build complicated contraptions to synthesize a specific gas, telecomms doesnt really have this- atleast to a major degree and it focus more on just repairing the telecomms system if its damaged as there isnt really much to build or tinker with. So I would like to put forward a few mechanics to add to telecomms to encourage the conduction of projects. (As this is a discussion feel free to criticize any of the ideas I put forward and contribute your own).

*Also note- even while I have played it a few times- I migrated to unitystation a long time ago so my familiarity with ss13 is hit or miss, I am going off the tgstation and ss13 wikis while writing this so I apologize for any mistakes)*

1) The ability for comm tech to add and create their own unique frequencies and tune headsets to them. Maybe by building an extra relay or bus mainframe, a comm tech can create there own frequency with a custom 3 letter code (e.g. [sec], [sci]) with a unique colour. The colour could either be from a list of options or a hexadecimal colour selector. This could allow a comm tech to create a unique frequency with there buddies or a new one by a request of some parties that wish to communicate more directly. I'm not sure about channel icons, but most likley all custom channels will just have a universal icon to represent that they are custom.

2) A tuning system, using a receiver- processor and mainframe hooked up to a special computer a comm tech will be able to scan frequencies to try find frequencies that match with channels. Lemme explain this further as that didnt make sense. Lets say, A syndicate invasion is expected, a comm tech may be asked to try tune into the syndicate frequency. The console will have a slider (like a radio) ranging form 1-1000MHz, every channel from command to syndicate will have a frequency alone this range, e.g 500MHz. If the console is tuned to a frequency that matches a channel, all messages on that channel will appear on the console logs.  If its close but doesnt match- the console will still print the logs, but they will be scrambled and unintelligable, much like an out of tune radio, this will help a player realize when they are near a channel to help them close in on it. This mechanic could also be used by lone syndicate comms officers to tune into and send messages on station channels, raising the skill level needed for this roles while maintaining there possibilities.

3) Radar maybe? this wouldnt be part of the sttion but could be a project a comm tech could under go, the tech could direct the radar dish in any direction, if something is in that direction, the tech would get a readout on there screen telling the distance to the object- and if the oject has a signiture- like a space ship- the signiture will show. This would also increase the value of stealth ships- they wouldnt produce a signature if the station spots them. (Unity's physics2D.raycast could do this quite easy)

The idea of all of this is to make telecomms just another aspect of enginnering along side and as fun to play with as construction electricity and atmos. Once again this is a discussion and I would love to hear ideas.

**Comment by mmails** [comment]
Sounds cool, someone make it a discussion

**Comment by MaxIsJoe** [comment]
I'm already working on adding the telecomms officer.
but we will not add him at all until we have a more robust chat system and all the signals PRs are added.

**Comment by CdrCross** [comment]
I would upgrade this to Network Tech and give the extra responsibilities of building and maintaining data servers for security records, medical records + cloning and research.
